thorlcr/activities/csvread/thcsvread.cpp

```diff
@@ -35,7 +35,21 @@ class CCsvReadActivityMaster : public CDiskReadMasterBase
         if (headerLines)
             mpTag = container.queryJob().allocateMPTag();
     }
-    void serializeSlaveData(MemoryBuffer &dst, unsigned slave)
+    virtual void validateFile(IDistributedFile *file) override
+    {
+        if (!isFileCsv(file))
```

Comment:
I don' think there is any harm in allowing csv to read any file format.  The main problem with mismatched files is when it causes thor to crash, and csv should never crash, regardless of the input.

Reply:
right, I'll allow CSV to read any type, will change to comment.